english/bob-barker/2050s/2050s.md

```diff
@@ -2,7 +2,7 @@ Your eyes flash open.  You realize you are late to something, but you are not su
 
 You look down at your wrist and see your iWrist flash red.
 
-A slight tingling sensation goes through your entire body as your iWrist sends a signal to your iMemory. 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.